### ISSUE TITLE
3d games

### ISSUE THREAD

**Issue by gedw99** [grow-graphics/eg#2]
hey @Splizard 

I was wondering if the golang library is capable of doing 3D games or quasi 3D ( isomorphic ).

**Comment by Splizard** [comment]
For sure, you can use the full power of Godot to manage assets and resources, 3D is certainly supported, we just don't have any examples yet.

**Comment by gedw99** [comment]
thanks @Splizard 

I need to dig into the golang package to see if there is an ECD system that we can do 3D with.

Then there is, I assume, the question of the 3D Model formats. This is what the Godot docs say:

glTF - supported in both text ( . gltf ) and binary ( . ...
DAE (Collada) - an older format that is still supported.
OBJ (Wavefront) - an older format that is supported, but the format is limited compared to modern options.
FBX - a commercial format that has limited support.

glTF looks like the best. If we will we need a golang glTF package in order to animate the models or not ? I assume we won't because the GoDot docs seems to indicate that 2D and 3D can be manipulated using the same API.

Interested in your thoughts on this..

**Comment by Splizard** [comment]
> If we will we need a golang glTF package in order to animate the models or not ?

I don't think you need any Go packages for this, Godot already has an excellent `.gltf` loader. I anticipate that you would import all of your models and setup levels and scenes within the Godot editor. Then you would be able to add behaviour by registering nodes in Go that you can add to your scenes.

**Comment by gedw99** [comment]
thanks for the info.

sounds like its worth a try :)

Need to find a simple GoDot project that has the original GLTF files to build an example from.
